<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>1041</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton_show_all_usr">
      <property name="geometry">
       <rect>
        <x>407</x>
        <y>160</y>
        <width>221</width>
        <height>36</height>
       </rect>
      </property>
      <property name="text">
       <string>Просмотр всех пользователей</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_crt_user">
      <property name="geometry">
       <rect>
        <x>433</x>
        <y>200</y>
        <width>171</width>
        <height>36</height>
       </rect>
      </property>
      <property name="text">
       <string>Создать пользователя</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_usr">
     <widget class="QListWidget" name="listWidget_people">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1021</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(50, 50, 50);
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_back_main_page_usr">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>360</y>
        <width>103</width>
        <height>36</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_crt_usr">
     <widget class="QLabel" name="label_login">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>80</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Логин:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_save">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>550</y>
        <width>191</width>
        <height>36</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_passw">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>130</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Пароль:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_login">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>70</y>
        <width>321</width>
        <height>36</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_pass">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>120</y>
        <width>321</width>
        <height>36</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_passw_2">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>175</y>
        <width>321</width>
        <height>36</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_passw_2">
      <property name="geometry">
       <rect>
        <x>503</x>
        <y>180</y>
        <width>171</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Подтвердите пароль:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_info">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Создание пользователя</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_back_main_page_crt_usr">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>550</y>
        <width>191</width>
        <height>36</height>
       </rect>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>ФИО:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_login_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>321</width>
        <height>36</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_login_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>175</y>
        <width>321</width>
        <height>36</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_login_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>321</width>
        <height>36</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_name_3">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>180</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Кабинет:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_name_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Должность:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
